cpp/cmake/thirdparty/get_mdspan.cmake

```diff
@@ -13,17 +13,19 @@
 # =============================================================================
 
 function(find_and_configure_mdspan VERSION)
-  rapids_cpm_find(
-    mdspan ${VERSION}
-    GLOBAL_TARGETS std::mdspan
-    BUILD_EXPORT_SET    raft-exports
-    INSTALL_EXPORT_SET  raft-exports
-    CPM_ARGS
-      GIT_REPOSITORY https://github.com/rapidsai/mdspan.git
-      GIT_TAG b3042485358d2ee168ae2b486c98c2c61ec5aec1
-      OPTIONS "MDSPAN_ENABLE_CUDA ON"
-              "MDSPAN_CXX_STANDARD ON"
-  )
+  if(RAFT_ENABLE_mdspan_DEPENDENCY)
```

Comment:
I'm curious why this `if` block is here instead of around the call to `find_and_configure_mdspan` below, as the other `get_*.cmake` files did.